screens/StepScreen.tsx

```diff
@@ -23,15 +25,16 @@ const StepScreen = (): JSX.Element => {
   const [stepAttempt, setStepAttempt] = useState<StepAttempt>();
   const [video, setVideo] = useState<AVPlaybackSource>();
   const chainMasteryState = useChainMasteryState();
+  const [isPLaying, setIsPlaying] = useState(false);
 
   /**
    * BEGIN: LIFECYCLE CALLS
    */
   useEffect(() => {
-    let isCancelled = false;
+    // let isCancelled = false;
```

Comment:
The checks for isCancelled are necessary to avoid unmounted component errors, which will cause a hard crash in the final built app. On the Android simulator, you can reliably trigger one of these errors by going through a training session, pausing the video, then navigating to the next step.

Reply:
Fixed in f5927f2e0a8fcaccfadf3e0c62f651ed75b8e988